packages/app/src/App.tsx

```diff
@@ -112,8 +113,29 @@ import { PuppetDbPage } from '@backstage/plugin-puppetdb';
 import { DevToolsPage } from '@backstage/plugin-devtools';
 import { customDevToolsPage } from './components/devtools/CustomDevToolsPage';
 import { CatalogUnprocessedEntitiesPage } from '@backstage/plugin-catalog-unprocessed-entities';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import Backend from 'i18next-http-backend';
+import { Translations } from '@backstage/core-plugin-api';
 
 const app = createApp({
+  initI18next: {
```

Comment:
This leaks a bit too much API from i18next straight into `createApp`. Likely the best way forward here is to add some layer in-between, for example something like this:

```
createApp({
  i18n: createBackstageI18nContext({ ... })
})
```

Where the i18n context is an opaque type that we can evolve independently. Let's leave that for a little bit later though.

Reply:
`modules: [LanguageDetector, Backend]` can be default options(Perhaps Backend isn't necessary depends on discussion above) in `AppTranslationApiImpl`

But `supportedLngs`, `resources` can not to be omitted even though use `createBackstageI18nContext({ ... })`